index.html

```diff
@@ -926,13 +954,18 @@ <h2>Client-Managed Version Creation</h2>
          <h2>Replacing Contents from Mementos</h2>
           <blockquote class="informative">
             Non-normative note:
-            Using the ingest-by-reference mechanism, one can replace the contents of an <a>LDPRv</a> with that of
-            an <a>LDPRm</a> by providing it's URL as the <code>URL</code> parameter in a <code>Content-Type:
-            message/external-body</code> header.  For example, given an <a>LDPRm</a> with URL <code>
-            http://example.org/some/memento</code>, the full header would be <br/>
-            <code>Content-Type: message/external-body; access-type=URL; expiration=1;<br/>
-            &nbsp;&nbsp;&nbsp;&nbsp;URL="http://example.org/some/memento"</code>
+            Using the ingest-by-reference mechanism provided by the <code>external-content-handling=copy</code>
```

Comment:
Above "copy" only say `requests that the server dereference the external content URI and treat that as if it were the entity body of the request.`   It does not say that it has to be ingested into the Fedora system.  So, "ingest-by-reference" doesn't quite hold here and this ultimately is different functionality.

Reply:
I think `handling="copy"` and POST combined _is_ ingest-by-reference so perhaps what I need to do is mention _"`POST` with `copy` external content handling"_?